src/caretogether-pwa/src/GeneratedClient.ts

```diff
@@ -4132,12 +4132,15 @@ export class Arrangement implements IArrangement {
     startedAtUtc?: Date | undefined;
     endedAtUtc?: Date | undefined;
     cancelledAtUtc?: Date | undefined;
+    plannedStartUtc?: Date | undefined;
+    plannedEndUtc?: Date | undefined;

```

Comment:
@LarsKemmann - what is the added value of the `| undefined;` declaration given that you're already using a `?` on the property?

Reply:
I'm not sure. _GeneratedClient.ts_ is auto-generated at compile time whenever the generated Swagger file changes.